src/components/map/MapService.js

```diff
@@ -935,15 +940,19 @@ goog.require('ga_urlutils_service');
           return $http.get(url).then(function(response) {
             // Live modifications for 3d test
             if (response.data) {
+              // pngjpeg is not supported by zeitreihen in 3D
               var ids = [
-                'ch.swisstopo.swissimage-product',
-                'ch.swisstopo.pixelkarte-farbe',
-                'ch.swisstopo.pixelkarte-grau',
                 'ch.swisstopo.zeitreihen'
               ];
               angular.forEach(ids, function(id) {
                 if (response.data[id]) {
-                  response.data[id].config3d = id + '_3d';
+                  var config3dId = id + '_3d';
+                  var config3d = angular.extend({},
+                      response.data[id], config3d);
+                  config3d.format = 'jpeg';
+                  config3d.useMapProxy = true;
+                  response.data[config3dId] = config3d;
+                  response.data[id].config3d = config3dId;
```

Comment:
Yes the issue comes from the `subLayersIds` where you to need to define an extra "fake" and empty layer to access.

Before we needed:

A config3d that references a *_3d layer
A *_3d layer that references other layers via subLayerIds (only for native layers)
A *.3d layer with the correct subLayerIds (real layers) or serverLayerName

Now:
A config3d that references a *_3d layer
A *_3d has all it needs (subLayersIds or serverLayerName)

I am not against the old behaviour, but It would require more DB changes.